Integrations/GitHub/GitHub.py

```diff
@@ -671,6 +672,28 @@ def get_commit_command():
     return_outputs(readable_output=human_readable, outputs=ec, raw_response=response)
 
 
+def list_commits(author):
+    params = {'author': author} if author else {}
+    return http_request('GET', url_suffix=f'{USER_SUFFIX}/commits',
+                        params=params)
+
+
+def list_commits_command():
+    args = demisto.args()
+    response = list_commits(author=args.get('author'))
+    limit = args.get('limit')
+    if limit:
+        response = response[:int(limit)]
+    commits = [format_commit_outputs(c['commit']) for c in response]
+    ec = {
+        'GitHub': {
+            'Commit': commits
+        }
```

Comment:
after you fix the outputs issue above, please replace this with the code:
```
ec = {
GitHub.Commit(val.SHA === obj.SHA)': commits
}
```
So it will merge the relevant objects in the context.